pypika/__init__.py

```diff
@@ -95,3 +96,6 @@
 __author__ = "Timothy Heys"
 __email__ = "theys@kayak.com"
 __version__ = "0.39.1"
+
+NULL = NullValue()
+SYSTEM_TIME = SystemTimeValue()
```

Comment:
There is a missing newline here. We still need to set up Black formatting at some point.